src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -310,6 +310,8 @@ $duo-tone-lighten: #000;
 
 .operating-status-flag {
   border-radius: 5px;
+  width: 286px;
+  height: 40px;
```

Comment:
Any concerns here about responsive design? Using fixed width can sometimes cause problems with that.

Reply:
it's ok. We have another issue to fix this later